spyplane/constants.py

```diff
@@ -22,3 +23,7 @@
 TICK_CHANNEL = int(os.getenv('PROD_TICK_DETECTION_CHANNEL_ID')) if flag_production else \
     int(os.getenv('TEST_TICK_DETECTION_CHANNEL_ID'))
 DB_PATH = './test_workspace/spyplane.db' if 'tests' in os.getcwd() else './workspace/spyplane.db'
+
+def log(msg):
+    print(msg)
+    sys.stdout.flush()
```

Comment:
You could just set flush=True on print, or map print to a partial with the keyword.  This reads that your end is likely not configured correctly though, as printing to stout should be flushing instantly without the need to force it.

Reply:
Thanks! I'm fixing it now